azurerm/internal/services/datafactory/data_factory_dataset_delimited_text_resource_test.go

```diff
@@ -59,7 +60,8 @@ func TestAccDataFactoryDatasetDelimitedText_http_update(t *testing.T) {
 				check.That(data.ResourceName).Key("schema_column.#").HasValue("2"),
 				check.That(data.ResourceName).Key("additional_properties.%").HasValue("1"),
 				check.That(data.ResourceName).Key("description").HasValue("test description 2"),
-				check.That(data.ResourceName).Key("compression_level").HasValue("Fastest"),
+				check.That(data.ResourceName).Key("compression_codec").HasValue("lz4"),
+				check.That(data.ResourceName).Key("compression_level").HasValue("Optimal"),
```

Comment:
`compression_level` is "Fastest" in the config here
```suggestion
				check.That(data.ResourceName).Key("compression_level").HasValue("Fastest"),
```